python/ql/src/experimental/Security/CWE-338/InsecureRandomness.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Insecure randomness
+ * @description Using insufficient randomness as the key of a cryptographic algorithm can allow an attacker to compromise security.
```

Comment:
I would like to keep descriptions consistent, so here I have copied the text from JS.

```suggestion
 * @description Using a cryptographically weak pseudo-random number generator to generate a
 *              security-sensitive value may allow an attacker to predict what value will
 *              be generated.
```